test/functional/apps/management/_import_objects.js

```diff
@@ -118,6 +118,7 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.settings.clickKibanaSavedObjects();
       await PageObjects.settings.importFile(path.join(__dirname, 'exports', '_import_objects_saved_search.json'));
       await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.settings.clickImportDone();
```

Comment:
Are these related to this change?

Reply:
not really. The test was passing locally but failing on CI. Closing the flyout allowed the test to pass locally and on CI

Reply:
Hmm, any idea how that's related to this?

Reply:
I have no idea. I looked into to it for the better part of Monday afternoon and and Tuesday morning. Must be timing related since it runs fine locally and only had problems in CI.